pages/linux/pmount.md

```diff
@@ -0,0 +1,15 @@
+# pmount
+
+> Mount arbitrary hotpluggable devices as normal user.
```

Comment:
This is the best online man page that I found
```suggestion
> Mount arbitrary hotpluggable devices as normal user.
> More information: <https://linux.die.net/man/1/pmount>.
```

Reply:
Ok, I thought only project homepages would count. Done.